src/components/DateInput/styles.tsx

```diff
@@ -0,0 +1,50 @@
+import styled from 'styled-components';
+import { Open_Sans as OpenSans } from 'next/font/google';
+import COLORS from '@/styles/colors';
+import { H4 } from '@/styles/text';
+
+const openSans = OpenSans({ subsets: ['latin'] });
+
+const getBorderColor = (error: boolean, filled: boolean): string => {
+  if (error) {
+    return COLORS.redMid;
+  }
+  if (filled) {
+    return COLORS.greyMid;
+  }
+  return COLORS.greyLight;
+};
+
+export const InputLabel = styled(H4)`
+  color: ${COLORS.greyDark};
+  margin-bottom: 0.625rem; //10px
+`;
+
+export const InputDate = styled.input<{ $error: boolean; $filled: boolean }>`
```

Comment:
i think we should add a `cursor: text` to the date input to stay consistent with other text-input-like designs